docker-compose/kuzzle-docker-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
       - NODE_ENV=production
 
   redis:
-    image: redis:3.2
+    image: redis:${REDIS_VERSION:-5}
```

Comment:
Why allowing to override the version on Redis only?

IMO, this is not needed at all, but if we go for it, we should include it for all services.

Reply:
Yes you're right :+1: